client/src/app/site/pages/meetings/pages/assignments/modules/assignment-poll/components/assignment-poll-vote/assignment-poll-vote.component.ts

```diff
@@ -180,14 +189,19 @@ export class AssignmentPollVoteComponent extends BasePollVoteComponent<ViewAssig
         return ``;
     }
 
-    public getVotesAvailable(user: ViewUser = this.user): number | string {
+    protected getVotesAvailable(user: ViewUser = this.user): number | string {
         if (this.isGlobalOptionSelected()) {
             return `-`;
         }
         return this.poll.max_votes_amount - this.getVotesCount(user);
     }
 
-    public async submitVote(user: ViewUser = this.user): Promise<void> {
+    protected override getVotesCount(user: ViewUser = this.user): number {
+        const boo = this.poll.isMethodY && this.poll.max_votes_per_option > 1 && !this.isGlobalOptionSelected(user);
```

Comment:
Don't call variables things like `boo`. Others won't be able to guess the intent behind them.